src/OrchardCore/OrchardCore.Deployment.Core/CommonPermissions.cs

```diff
@@ -0,0 +1,15 @@
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Deployment
+{
+    /// <summary>
+    /// This class contains the source references to the OrchardCore.Deployment module permissions so they can be used in other modules
+    /// without having to reference the OrchardCore.Deployment by itself.
+    /// </summary>
+    public class CommonPermissions
+    {
+        public static readonly Permission ManageDeploymentPlan = new Permission("ManageDeploymentPlan", "Manage deployment plan");
```

Comment:
`ManageDeploymentPlan` => `ManageDeploymentPlans` may be

Reply:
Hmm, could be, I haven't changed this (just moved it). Could just update the description, without changing the permission